e2e/test/scenarios/onboarding/setup/setup.cy.spec.js

```diff
@@ -287,19 +287,20 @@ describeWithSnowplow("scenarios > setup", () => {
   });
 
   it("should send snowplow events", () => {
-    // 1 - pageview
+    // 1 - new_instance_created
```

Comment:
see https://github.com/metabase/metabase/pull/23893#discussion_r921029709 for why we made this change in the first place

Reply:
Gotcha, thanks!